iotqatools/cb_ngsiv2_utils.py

```diff
@@ -0,0 +1,371 @@
+                 path_update_or_append_entity_attributes="/v2/entities/entityId/attrs",
+                 path_update_existing_entity_attributes="/v2/entities/entityId/attrs",
+                 path_replace_all_entity_attributes="/v2/entities/entityId/attrs",
+                 path_remove_entity="/v2/entities",
+                 path_get_attribute_data="/v2/entities/entityId/attrs/attrName",
+                 path_update_attribute_data="/v2/entities/entityId/attrs/attrName",
+                 path_remove_a_single_attribute="/v2/entities/entityId/attrs/attrName",
+                 path_get_attribute_value="/v2/entities/entityId/attrs/attrName/value",
+                 path_attribute_value_update="/v2/entities/entityId/attrs/attrName/value",
+                 path_retrieve_entity_types="/v2/types",
+                 path_retrieve_emtity_type="/v2/types/entityType",
+                 path_retrieve_subscriptions="/v2/subscriptions",
+                 path_retrieve_subscription_by_id="/v2/subscriptions/subscriptionId",
+                 path_update_subscription="/v2/subscriptions/subscriptionId",
+                 path_delete_subscription="/v2/subscriptions/subscriptionId",
+                 path_statistics="/statistics",
+                 path_version="/version",
+                 log_instance=None,
+                 log_verbosity='DEBUG',
+                 default_headers={'Accept': 'application/json'},
+                 check_json=True):
+        """
+        CB Utils constructor
+        :param instance:
+        :param protocol:
+        :param port:
+        :param path_list_entities:
+        :param path_create_entity:
+        :param path_retrieve_entity_by_id:
+        :param path_retrieve_entity_attributes:
+        :param path_update_or_append_entity_attributes:
+        :param path_update_existing_entity_attributes:
+        :param path_replace_all_entity_attributes:
+        :param path_remove_entity:
+        :param path_get_attribute_data:
+        :param path_update_attribute_data:
+        :param path_remove_a_single_attribute:
+        :param path_get_attribute_value:
+        :param path_attribute_value_update:
+        :param path_retrieve_entity_types:
+        :param path_retrieve_emtity_type:
+        :param path_retrieve_subscriptions:
+        :param path_retrieve_subscription_by_id:
+        :param path_update_subscription:
+        :param path_delete_subscription:
+        :param path_statistics:
+        :param path_version:
+        :param log_instance:
+        :param log_verbosity:
+        :param default_headers:
+        :param check_json:
+        """
+        # initialize logger
+        if log_instance is not None:
+            self.log = log_instance
+        else:
+            self.log = get_logger('CbNgsi10Utilsv2', log_verbosity)
+
+        # Assign the values
+        self.default_endpoint = protocol + '://' + instance + ':' + port
```

Comment:
I would avoid the summatory strings, instead of this: 
`self.default_endpoint = protocol + '://' + instance + ':' + port`
use 
`self.default_endpoint = "{}://{}:{} ".format(protocol, instance + port)